src/services/namex-services.ts

```diff
@@ -122,6 +122,20 @@ export default class NamexServices {
     }
   }
 
+  /** Searches LEAR (Legal API) for business record for specified corp num. */
+  static async searchLear (corpNum: string): Promise<any> {
+    const url = `${appBaseURL}/businesses/${corpNum}`
+    return this.axios.get(url).then(response => response.data?.business)
+  }
```

Comment:
This is a proxied call (Namerequest UI <--> Namex API <--> Legal API) and it returns the same object as `BusinessServices.fetchBusiness()`.